modules/ROOT/pages/xml-sdk.adoc

```diff
@@ -1482,31 +1545,111 @@ GitHub Location: https://github.com/mulesoft-labs/smart-connectors-integration-t
-                    'Wubba Lubba Dub Dub'
-            ]]></ee:set-payload>
+                    {
+                        'name' : 'Rick',
+                        'lastname' : 'Sanchez'
+                    }
+                    ]]></ee:set-payload>
+                </ee:message>
+            </ee:transform>
+        </body>
+        <output type="PersonJsonType" doc:description="Payload's output"/>
+    </operation>
+
+    <operation name="set-payload-through-dw-fun" doc:description="Sets the payload to the structure of PersonJsonType through a function call">
+        <body>
+            <ee:transform>
+                <ee:message>
+                    <ee:set-payload><![CDATA[
+                    %dw 2.0
+                    output application/json encoding='UTF-8'
+                    ---
+                    {
+                        'number' : module_using_dw::weirdFolder::dwModule::five()
+                    }
+                    ]]></ee:set-payload>
+                </ee:message>
+            </ee:transform>
+        </body>
+        <output type="number" doc:description="Payload's output"/>
+    </operation>
+
+    <operation name="set-payload-through-dw-fun-with-string-type" doc:description="Sets the payload to the parameterized string 'name' through a function call passing through parameters">
+        <parameters>
+            <parameter name="name" type="string"/>
+        </parameters>
+        <body>
+            <ee:transform>
+                <ee:message>
+                    <ee:set-payload><![CDATA[
+                    %dw 2.0
+                    output application/json encoding='UTF-8'
+                    ---
+                        module_using_dw::weirdFolder::dwModule::echo(vars.name)
+                    ]]></ee:set-payload>
+                </ee:message>
             </ee:transform>
         </body>
         <output type="string" doc:description="Payload's output"/>
     </operation>
+
+    <operation name="set-payload-through-dw-fun-with-person-type" doc:description="Sets the payload to the structure of PersonJsonType through a function call passing through parameters">
+        <parameters>
+            <parameter name="person" type="PersonJsonType"/>
+        </parameters>
+        <body>
+            <ee:transform>
+                <ee:message>
+                    <ee:set-payload><![CDATA[
+                    %dw 2.0
+                    output application/json encoding='UTF-8'
+                    ---
+                        module_using_dw::weirdFolder::dwModule::toUpper(vars.person)
+                    ]]></ee:set-payload>
+                </ee:message>
+            </ee:transform>
+        </body>
+        <output type="PersonJsonType" doc:description="Payload's output uppercased"/>
+    </operation>
+
+    <operation name="set-payload-through-dw-fun-with-persons-type" doc:description="Sets the payload to the structure of PersonsJsonType through a function call passing through parameters">
```

Comment:
Is this section from <operation name="set-payload..." the same as the one right above it?

Reply:
Yup! One is PersonJsonType and one is PersonsJsonType. Thank you for checking!